<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="joy" type="joy_node" name="joy_node" output="screen"/>
  <node pkg="memmo_teleop" type="memmo_teleop_joystick" name="memmo_teleop_joystick" output="screen"/>
</launch>
